src/lambdaDestination.ts

```diff
@@ -0,0 +1,17 @@
+import * as lambda from "@aws-cdk/aws-lambda";
+import * as logs from "@aws-cdk/aws-logs";
+import { Construct } from "@aws-cdk/core";
+
+/**
+ * Use a Lamda Function as the destination for a log subscription.
+ * Using this class temporarily until https://github.com/aws/aws-cdk/pull/14222 is merged and released.
+ */
+export class LambdaDestination implements logs.ILogSubscriptionDestination {
+  constructor(private readonly fn: lambda.IFunction) {}
+
+  public bind(scope: Construct, logGroup: logs.ILogGroup): logs.LogSubscriptionDestinationConfig {
+    scope.toString;
+    logGroup.logGroupName; //get around not using the params
```

Comment:
What does this comment mean?

Reply:
basically if I didn't use the scope or logGroup parameters the compiler would complain so I added some pretty much otherwise useless code (line 13, 14) so that the compiler wouldn't complain.

Reply:
Also so that I can implement the [bind](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-logs/lib/subscription-filter.ts) function correctly on the `ILogSubscriptionDestination` interface.

Reply:
On second thought I might be able to just remove the scope and logGroup from the bind function.

Reply:
yeah I just removed the scope and logGroup params, I was under the impression I had to keep it that way to implement the interface correctly.